### ISSUE TITLE
Nested create is not working

### ISSUE THREAD

**Issue by pantharshit00** [prisma/photonjs#33]
Might be related to #30 

Given this datamodel:
```grovvy
datasource db {
  provider = "sqlite"
  url      = "file:dev.db"
  default  = true
}

generator photon {
  provider = "photonjs"
}

model User {
  id       Int    @id
  username String @unique
  posts    Post[]
}

model Post {
  id   Int    @id
  data String
  user User
}
```

This mutation doesn't work:

```js
 const testData = await photon.users.create({
    data: {
      username: 'harshit',
      posts: {
        create: {
          data: 'test',
        },
      },
    },
  })
```

Reproduction repo: https://github.com/pantharshit00/p2-nested-create

**Comment by matthewmueller** [comment]
Still present

**Comment by pantharshit00** [comment]
Blocked . because of #177. I can't post error because of it

**Comment by pantharshit00** [comment]
Fixed in latest alpha:`prisma2@2.0.0-alpha.94, binary version: 026ffceadcf2b8c42e6b9568a90ba48bbf7c47f1`
